src/Elcodi/Admin/BannerBundle/Controller/BannerZoneController.php

```diff
@@ -103,7 +103,7 @@ public function listAction(
      * @return array Result
      *
      * @Route(
-     *      path = "s/list/component/{page}/{limit}/{orderByField}/{orderByDirection}",
+     *      path = "s/component/{page}/{limit}/{orderByField}/{orderByDirection}",
```

Comment:
imo `/components/` should be used here instead of `/component/`, so it's easy to understand that's a list
(the same for every case below)